geode-core/src/main/resources/org/apache/geode/internal/sanctioned-geode-core-serializables.txt

```diff
@@ -457,6 +457,12 @@ org/apache/geode/management/internal/functions/CacheRealizationFunction,false
 org/apache/geode/management/internal/functions/CliFunctionResult$StatusState,false
 org/apache/geode/management/internal/functions/GetMemberInformationFunction,true,1404642539058875565
 org/apache/geode/management/internal/functions/RebalanceFunction,true,1
+org/apache/geode/management/internal/functions/RestoreRedundancyFunction,true,1
+org/apache/geode/management/runtime/RestoreRedundancyResponse,true,1
```

Comment:
I believe some of these belongs to sanctioned-geode-management-serializables.txt

Reply:
Great catch. This is one of the changes that I am just not sure about. I put in a placeholder values that allow it to work, but I need to really consider the actual values.